eslint-config-base/index.js

```diff
@@ -102,6 +102,14 @@ module.exports = {
       message: "`with` is disallowed in strict mode because it makes code impossible to predict and optimize.",
     }],
 
+    // Leaving out the await can hide the fact that you're not catching thrown errors:
+    // https://twitter.com/_jayphelps/status/1324565522755788803
+    // Additionally, it will prevent the function from being part of the stack trace
+    // if not catching them:
+    // https://eslint.org/docs/rules/no-return-await
+    // This does not outweigh the minor performance benefit, in most cases.
+    "no-return-await": ["off"],
```

Comment:
```suggestion
    "no-return-await": ["off"],
    // https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/return-await.md
    "@typescript-eslint/return-await": "in-try-catch"
```

Better yet, we're not the first to think about that, [TypeScript eslint accomodates it](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/return-await.md)!

See also: https://github.com/typescript-eslint/typescript-eslint/issues/994

Reply:
Love it :)